src/content-tags-drawer/ContentTagsDrawer.tsx

```diff
@@ -0,0 +1,390 @@
+import React, { useContext, useEffect } from 'react';
+import {
+  Container,
+  Spinner,
+  Stack,
+  Button,
+  Toast,
+} from '@openedx/paragon';
+import { useIntl, FormattedMessage } from '@edx/frontend-platform/i18n';
+import { useParams, useNavigate } from 'react-router-dom';
+import classNames from 'classnames';
+import messages from './messages';
+import ContentTagsCollapsible from './ContentTagsCollapsible';
+import Loading from '../generic/Loading';
+import useContentTagsDrawerContext from './ContentTagsDrawerHelper';
+import { ContentTagsDrawerContext, ContentTagsDrawerSheetContext } from './common/context';
+
+interface TaxonomyListProps {
+  contentId: string;
+}
+
+const TaxonomyList = ({ contentId }: TaxonomyListProps) => {
+  const navigate = useNavigate();
+  const intl = useIntl();
+
+  const {
+    isTaxonomyListLoaded,
+    isContentTaxonomyTagsLoaded,
+    tagsByTaxonomy,
+    stagedContentTags,
+    collapsibleStates,
+  } = React.useContext(ContentTagsDrawerContext);
+
+  if (isTaxonomyListLoaded && isContentTaxonomyTagsLoaded) {
+    if (tagsByTaxonomy.length !== 0) {
+      return (
+        <div>
+          { tagsByTaxonomy.map((data) => (
+            <div key={`taxonomy-tags-collapsible-${data.id}`}>
```

Comment:
Nit: React `key`s aren't rendered into the DOM and don't have to be globally unique; just unique within this parent element. So `<div key={data.id}>` would be perfectly fine.

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1299/commits/ff7bd2a751bab3a8ddf53b37bdeb44248f97ef5e